### PR TITLE
Adding profile information to Awsaml

### PR DESCRIPTION
**Pull request by dgreene-r7**
This PR addresses issue #4 and adds instructions (platform specific) for exporting environment variables so users can copy/paste directly to their terminal of choice and set their profile.

### CODE REVIEW

**Comment by fmitchell-r7** [comment]
This looks really good!

We probably want to bump the [default height](https://github.com/rapid7/awsaml/blob/master/app.js#L10) up to 800 to account for the extra vertical space the commands take up.

File: views/refresh.jsx [hunk 1]
```diff
@@ -19,10 +33,25 @@ class Refresh extends React.Component { // eslint-disable-line react/display-nam
...
+              <dd>{this.props.accessKey}</dd>
+              <dt>Secret Key:</dt>
+              <dd>{this.props.secretKey}</dd>
+              <dt>Session Token:</dt>
+              <dd>{this.props.sessionToken}</dd>
+            </dl>
+            <div className='col-centered rounded-12 content env-var'>
+              <p>Run these commands from a {this.term} to use the AWS CLI:</p>
+              <pre>
+                <code className='language-powershell'>
```
Comment (fmitchell-r7) [inline views/refresh.jsx]: Do we want to switch the syntax highlight based on `platform` as well?

Comment (dgreene-r7) [inline views/refresh.jsx]: I played around with it. The powershell class looks better than the bash one.

**Comment by fmitchell-r7** [comment]
:+1: Looks good to me.

**Comment by dgreene-r7** [comment]
Resolve #4

**Pull request merged**